CHANGELOG.md

```diff
@@ -2,6 +2,95 @@
 
 ## 28
 
+### Call Caching
```

Comment:
I'll reiterate my general stance that changelog entries should be kept to a few lines at most